test/query_test.rb

```diff
@@ -363,6 +378,22 @@ def test_modulo
     assert_equal 2, client.query { modulo(2) }
   end
 
+  def test_lt
+    assert_equal true, client.query { lt 1, 2 }
```

Comment:
We only need to assert enough that the form is being submitted correctly. Remember we don't need to test the DB's behavior in any of these tests.